yarn.lock

```diff
@@ -2415,16 +2415,6 @@
     regexpp "^3.0.0"
```

Comment:
Avoid lock files changes from the PR

Reply:
seems the lock file is not up to date. I had a fresh installation before starting the work.

Reply:
Okay